src/api/goods.js

```diff
@@ -6,6 +6,11 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => getAll()
+  .then(goods => (
+    goods.sort((a, b) => a.name.localeCompare(b.name))
+      .slice(0, 5)
```

Comment:
If you chain array methods, try to keep them all on the same level:

```suggestion
    goods
      .sort((a, b) => a.name.localeCompare(b.name))
      .slice(0, 5)
```